interfaces/builtin/cpu_control.go

```diff
@@ -82,6 +82,11 @@ const cpuControlConnectedPlugAppArmor = `
 
 # Allow control CPU C-states switching see: https://docs.kernel.org/power/pm_qos_interface.html#pm-qos-framework
 /dev/cpu_dma_latency rw,
+
+# Allow interrupt affinity settings, see https://www.kernel.org/doc/html/latest/core-api/irq/irq-affinity.html
+/proc/interrupts r,
+/proc/irq/[0-9]+/smp_affinity rw,
```

Comment:
should we also allow `/proc/irq/default_smp_affinity` ?

Reply:
Yeah! It's a good idea! Thanks for the suggestion!

Reply:
Added on https://github.com/canonical/snapd/pull/14424/commits/8cce4658160770ae3677181707faa36d26587a51. Thank you!